source/_integrations/esphome.markdown

```diff
@@ -50,6 +50,25 @@ This integration allows [ESPHome](https://esphome.io) devices to connect directl
 
 {% include integrations/config_flow.md %}
 
+### Required manual input
+
+To configure an ESPHome device, enter the following information:
+
+{% configuration_basic %}
+
+host:
+  description: "IP address or hostname of the ESPHome device. <br> This will be pre-filled if the device was auto-discovered."
+port:
+  description: "Port used by the ESPHome native API. <br> This will be pre-filled if the device was auto-discovered."
```

Comment:
```suggestion
  description: "Port used by the ESPHome native API (default: 6053). <br> This will be pre-filled if the device was auto-discovered."
```